core/src/main/java/org/elasticsearch/common/network/IfConfig.java

```diff
@@ -34,13 +34,13 @@
 /**
  * Simple class to log {@code ifconfig}-style output at DEBUG logging.
  */
-final class IfConfig {
+public final class IfConfig {
 
     private static final Logger logger = Loggers.getLogger(IfConfig.class);
     private static final String INDENT = "        ";
 
     /** log interface configuration at debug level, if its enabled */
-    static void logIfNecessary() {
+    public static void logIfNecessary() {
         if (logger.isDebugEnabled()) {
             try {
```

Comment:
I was curious about this `try` block and I note that it currently catches `SecurityException` which should not be needed anymore with this change. Can we remove it now?